soap_api/mastr_power_unit_download.py

```diff
@@ -73,8 +73,7 @@ def get_power_unit(start_from, limit=API_MAX_DEMANDS):
 
 def download_power_unit(
         power_unit_list_len=2363200,
-        limit=API_MAX_DEMANDS,
-        ofname=None
```

Comment:
Please keep the option to give the ofname directly, the tests need it and the user is always free **not** to use it